en/docs/develop-components/develop-components-with-git.md

```diff
@@ -48,6 +48,13 @@ Authorizing Choreo as a Bitbucket application provides Choreo the following perm
 |Pull Requests | Y   | Y    | Create a pull request if you have chosen to start with a Choreo sample |
 |Webhooks      | Y   | Y    | Trigger automatic deployment and configuration generation              |
 
+## Authorize GitLab Self-Managed with Choreo
 
+Authorizing using a Personal Access Token (PAT) obtained from your GitLab Self-Managed Server provides Choreo with the following permissions to perform the respective actions on your behalf within the repository.
```

Comment:
Same suggestion as above on capitalization.

Reply:
Addressed in https://github.com/YashodhaSamarawickrama/docs-choreo-dev/pull/1